network-traversal-quickstart/javascript/getRelayConfiguration.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @summary Issue a new Relay configuration
+ */
+
+const { CommunicationIdentityClient } = require("@azure/communication-identity");
+const { CommunicationRelayClient } = require("@azure/communication-network-traversal");
+
+// Load the .env file if it exists
+const dotenv = require("dotenv");
+dotenv.config();
+
+// You will need to set this environment variables or edit the following values
+const connectionString =
+  process.env["COMMUNICATION_CONNECTION_STRING"] || "<communication service connection string>";
+
+async function main() {
+  console.log("\n== Get Relay configuration Sample ==\n");
+
+  const identityClient = new CommunicationIdentityClient(connectionString);
+
+  // Create user
+  console.log("Creating User");
+
+  const user = await identityClient.createUser();
+
+  const relayClient = new CommunicationRelayClient(connectionString);
+  console.log("Getting relay configuration");
+
+  const config = await relayClient.getRelayConfiguration(user);
```

Comment:
Ideally we'd actually use the relay configuration in the most basic way, like setting up a [datachannel](https://www.html5rocks.com/en/tutorials/webrtc/datachannels/). We don't need to code the entire E2E scenario, but just showing how simple it is in JS to put the given relay config into a `RTCPeerConnection` would be helpful

Reply:
I can add it as a comment. To use run/use RTCPeerConnection the JS code needs to be executed on a browser. Also we can code a node server as in the hero sample. But if we want to keep it simple, I don't think we can show the RTCPeerConnection creation, it will throw "RTCPeerConnection" is undefined.

Reply:
I'm surprised it's not possible to use WebRTC unless you run JS in a browser, but if we want to have the code as comment, we're not actually using the `CommunicationRelayConfiguration` gained from `await relayClient.getRelayConfiguration()`

Reply:
Well, it's not like its not possible. As I mentioned it can be done using a node server and seems there are other alternatives. But I was surprised too I couldn't just use it. I've been searching on how to do that without adding more complexity since this is a QuickStart for NetworkTraversal and I haven't find how. There are some options but they end up introducing code that we probably don't want to have in this sample.

Here they mentioned that you need the browser. https://stackoverflow.com/questions/62350255/how-to-use-webrtc-apis-in-node-js-without-browser-or-any-html

Here are some alternatives mentioned: https://stackoverflow.com/questions/16429862/creating-a-webrtc-peer-without-a-browser-with-just-a-javascript-interpreter